src/calculateRentalCost.js

```diff
@@ -3,8 +3,24 @@
  *
  * @return {number}
  */
+
 function calculateRentalCost(days) {
-  // write code here
+  const DAY_RENT = 40;
+  const SHORT_TERM = 3;
+  const LONG_TERM = 7;
+  let discount = 0;
+
+  if (days >= SHORT_TERM && days < LONG_TERM) {
+    discount = -20;
+  }
+
+  if (days >= LONG_TERM) {
+    discount = -50;
+  }
+
+  const TOTAL_RENT = days * DAY_RENT + discount;
```

Comment:
why is it a CONSTANT?
u should look at the difference between a `variable` and a `CONSTANT` - first comment